tests/helpers.py

```diff
@@ -215,8 +224,16 @@ def create_pvc(sc_name, pvc_name=None):
         OCS: An OCS instance for the PVC
     """
     pvc_data = copy.deepcopy(defaults.CSI_PVC_DICT)
+    if 'rbd' in sc_name:
```

Comment:
that's not a good way to determine if that's a rbd or cephfs.
Simply pass the interface_type to the function

Reply:
@shreekarSS  @ebenahar I have one doubt. In helpers.create_storage_class, we are using interface_type and interface_name. 
```
create_storage_class(
    interface_type, interface_name, secret_name, sc_name=None
):
...
 sc_data['parameters']['pool'] = interface_name
```

Since, both CephFS and RBD storageclaasses have similar **parameter** name called **pool** (i.e  sc_data['parameters']['pool'] ) , would it be more convenient to call **"interface_name**" as "**pool_name**" .

Please suggest

Reply:
I think 'pool_name' would be  more convenient @nehaberry

Reply:
> that's not a good way to determine if that's a rbd or cephfs.
> Simply pass the interface_type to the function

I've modified create_pvc() to have interface_type , changing the files impacted